src/main/java/launcher/Lehrry.java

```diff
@@ -1,7 +1,23 @@
 package launcher;
 
+import listeners.MainEventListener;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.JDABuilder;
+import net.dv8tion.jda.api.requests.GatewayIntent;
+import net.dv8tion.jda.api.utils.ChunkingFilter;
+
+import javax.security.auth.login.LoginException;
+
 public class Lehrry {
     public static void main(String[] args) {
-        System.out.println("hey!");
+        try {
+            JDA jda = JDABuilder.createDefault(System.getenv("LEHRRY_TOKEN"))
+                    .setChunkingFilter(ChunkingFilter.ALL)
```

Comment:
If you want the member list to be updated with all of the members of the guild, you need to add .setMemberCachePolicy(MemberCachePolicy.ALL) or something like that. You can check Auther for the exact code

Reply:
I don't believe my code accesses the guild's member list at any point, so this would only add unnecessary event receipts.